src/librbd/Journal.h

```diff
@@ -109,6 +109,9 @@ class Journal {
   static int request_resync(ImageCtxT *image_ctx);
   static int promote(ImageCtxT *image_ctx);
 
+  static void init_safetimer_instance(CephContext *cct, SafeTimer **timer,
```

Comment:
Minor: I think this will be re-usable elsewhere in librbd, so can you move it to ImageCtx and rename it ```get_timer_instance()```?